core/examples/luigi-sample-angular/package.json

```diff
@@ -37,16 +38,23 @@
     "@angular/cli": "^6.1.1",
     "@angular/compiler-cli": "^6.1.0",
     "@angular/language-service": "^6.1.0",
+    "@babel/core": "^7.0.0",
+    "@babel/plugin-proposal-class-properties": "^7.2.3",
+    "@babel/preset-env": "^7.2.3",
+    "@babel/preset-es2017": "^7.0.0-beta.53",
+    "@babel/register": "^7.0.0",
     "@types/jasmine": "~2.8.6",
     "@types/jasminewd2": "~2.0.2",
     "@types/node": "^6.0.115",
     "babel-loader": "^8.0.4",
     "babel-preset-minify": "^0.5.0",
     "codelyzer": "^4.4.1",
+    "concurrently": "^4.1.0",
```

Comment:
We talked about using concurrently instead of native background process spawning. Since I am not sure if my method works on Windows, I am fine to use such a module that does it for us.
FYI, if you want to spawn something in the background in mac/linux, you just add an ampersand at the end. 
`"start": "(npm run buildConfig &) && ng serve",`

Reply:
Ohh, you're right! I didn't know about `(... &)` trick. I thought you were talking about `&&` which wouldn't work standalone.